.github/workflows/games.yml

```
@@ -86,60 +86,117 @@ jobs:
+          set -eo pipefail
           chmod +x new-engine/engine
           chmod +x base-engine/$OUTPUT_EXEC
+          chmod +x tools/fastchess
+          chmod +x tools/ordo-linux64
+          tools/fastchess \
+            -engine cmd=new-engine/engine name=new \
+            -engine cmd=base-engine/$OUTPUT_EXEC name=base \
+            -openings file=tools/UHO_Lichess_4852_v1.epd format=epd order=random \
+            -each tc=$TC -startup-ms 600000 -ucinewgame-ms 600000 -ping-ms 600000 \
+          ROUNDS: ${{ github.event.inputs.rounds }}
+        run: |
```

actionlint (1.7.12)

[error] key "run" is duplicated in element of "steps" section. previously defined at line:127,col:9

(syntax-check)